[2025-03-24T13:21:34.299+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T13:21:34.344+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-03-21T00:00:00+00:00 [queued]>
[2025-03-24T13:21:34.366+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-03-21T00:00:00+00:00 [queued]>
[2025-03-24T13:21:34.367+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T13:21:34.397+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-21 00:00:00+00:00
[2025-03-24T13:21:34.523+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=366) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T13:21:34.526+0000] {standard_task_runner.py:72} INFO - Started process 378 to run task
[2025-03-24T13:21:34.526+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-03-21T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmprsp1phhe']
[2025-03-24T13:21:34.528+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask first_task
[2025-03-24T13:21:34.637+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-03-21T00:00:00+00:00 [running]> on host e1537a8d388b
[2025-03-24T13:21:34.876+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T00:00:00+00:00'
[2025-03-24T13:21:34.877+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T13:21:34.914+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T13:21:34.916+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is my first task']
[2025-03-24T13:21:34.941+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T13:21:34.943+0000] {subprocess.py:106} INFO - hello world, this is my first task
[2025-03-24T13:21:34.944+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T13:21:34.996+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T13:21:34.997+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-03-21T00:00:00+00:00, execution_date=20250321T000000, start_date=20250324T132134, end_date=20250324T132134
[2025-03-24T13:21:35.073+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T13:21:35.120+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-24T13:21:35.123+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
